### PR TITLE
[FEAT] standardise names for functions

### PR DESCRIPTION
**Pull request by MOchiara**

### CODE REVIEW

**Comment by MOchiara** [comment]
We should rename maybe these two but I am not sure how.
*  tools.plot_vertical_speeds_with_histograms
* tools.plot_combined_velocity_profiles

**Comment by callumrollo** [comment]
I think those are fine names! Doesn't matter if they're long, they are descriptive

**Comment by MOchiara** [comment]
This will resolve #99

File: notebooks/demo.ipynb [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "import numpy as np\n",
     "from glidertest import fetchers\n",
     "from glidertest import tools\n",
-    "import warnings\n",
+    "#import warnings\n",
```
Comment (callumrollo) [inline notebooks/demo.ipynb:32]: we can probably get rid of this warning mute now that you've fixed it in the tools themselves?

**Review by callumrollo — APPROVED** [review]

**Pull request merged**